cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -513,7 +513,7 @@ class fixed_point_column_wrapper : public detail::column_wrapper {
    * @code{.cpp}
    * // Creates a non-nullable column of INT32 elements with 5 elements: {0, 2, 4, 6, 8}
```

Comment:
Is this correct? It creates a fixed-point column, not an INT32 column.  Should be `DECIMAL32`?

Reply:
Correct. I fixed the docs here where it was a copy/paste from elsewhere and missed this. Changed it to `DECIMAL32`. The underlying storage in `int32_t`, but the column type is `DECIMAL32`.